lib/shared/drive-constraints.js

```diff
@@ -376,9 +377,10 @@ exports.getDriveImageCompatibilityStatuses = (drive, image) => {
       message: exports.COMPATIBILITY_STATUS_MESSAGES.LOCKED
     })
   } else if (!_.isNil(drive) && !_.isNil(drive.size) && !exports.isDriveLargeEnough(drive, image)) {
+    const imageSize = _.get(image, [ 'size', 'final', 'estimation' ]) ? image.size.final.value : image.size.original
```

Comment:
This logic looks weird. Basically, if the image final size is an estimation, use that, otherwise use the original one, which means we will use the original size even for compressed sizes that we know their final size for sure (i.e. that are not estimations).

Maybe the intended logic is: If the final one is *not* an estimation, use that, otherwise use the original size.

Reply:
Oh boy, yeah, I messed that up – fixed now :)

Reply:
> Maybe the intended logic is: If the final one is not an estimation, use that, otherwise use the original size.

Isn't that exactly what the 'estimated' size is anyway?